### PR TITLE
Migrate Skill Detail Page, child components to Chat SUSI #2134

### LINKED ISSUE #2134: Migrate CMS into Chat Repo
**Actual Behaviour**
* Currently they are 2 separate repos

**Expected Behaviour**
* CMS should be integrated to this repo, under /skills route

- [x] Move Home Page and category, language pages
- [x] Move Skill details pages
- [ ] Move Botbuilder
- [ ] Move Skill creator, Skill editor
- [ ] Move Skill History, version
- [ ] Dashboard
- [ ] Admin

**Would you like to work on the issue?**
Yes. Also, @fragm3 , @harsh-jindal

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2134, #2140

Changes: Migrated Skill Detail Page, child components to Chat SUSI 

Demo Link: 
- https://pr-2139-fossasia-susi-web-chat.surge.sh
- Fix links in skills page #2140

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -107,6 +109,16 @@ class App extends Component {
                 path="/skills/language/:language"
                 component={BrowseSkillByLanguage}
               />
+              <Route
+                exact
+                path="/:category/:skill/:lang"
+                component={SkillListing}
+              />
```
Comment (akshatnitd) [inline src/App.js]: Path should start with `/skills`

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Comment by fragm3** [comment]
@akshatnitd still uploading will edit this message as soon it gets deployed.

**Comment by harsh-jindal** [comment]
@fragm3 I think correct link is https://pr-2139-fossasia-susi-web-chat.surge.sh/

**Comment by fragm3** [comment]
@harsh-jindal I am still deploying, link you have provided works fine 🤣@akshatnitd do you also deploy it by building and typing surge?

**Comment by harsh-jindal** [comment]
@fragm3 It is auto deployed right. I just add the pr number in the link given in the pr template works fine for me always. Whats the logic of -1- ?

**Comment by fragm3** [comment]
@harsh-jindal funny, I used to manually deploy it 🤣, editing the link

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal please review, link: https://pr-2139-fossasia-susi-web-chat.surge.sh

**Comment by harsh-jindal** [comment]
All routes seems to working good. 
After this is merged I'll work on the rest.

**Comment by fragm3** [comment]
@harsh-jindal go ahead with botbuilder, skillcreator, dashboard component

**Comment by akshatnitd** [comment]
@fragm3 I know that it is automatically deployed as I myself had worked on it :D

File: public/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.2/leaflet.css" async/>
     <script src="https://www.google.com/recaptcha/api.js?render=explicit" async defer></script> 
+    <script src="https://www.gstatic.com/charts/loader.js" async defer></script>
```
[the remark this answers is not in the document]
Comment (fragm3) [reply public/index.html]: It was required for charts to work, gave error otherwise.

**Comment by fragm3** [comment]
@akshatnitd I wish I knew that earlier 😞 Made the changes

**Pull request merged**